src/components/ux/SampleArrow.jsx

```diff
@@ -43,7 +43,7 @@ const SampleArrow = (props) => {
 
   return (
     <div
-      className={className}
+      className={`${className} flex justify-center items-center border-primary-blue`}
       style={customStyle()}
       onClick={onClick}
       onKeyDown={onClick}
```

Comment:
Rather than embedding the class names directly within this UX component, incorporate them at the point where the component is invoked. The `className` prop is designed specifically for this use.

For instance: `<SampleArrow ariaLabel="Next" className='flex justify-center items-center border-primary-blue' />,`